### PR TITLE
Package ocsigen-start.1.6.0

### PR DESCRIPTION
**Pull request by jrochel**
### `ocsigen-start.1.6.0`
An Eliom application skeleton ready to use to build your own application with users, (pre)registration, notifications, etc
Ocsigen Start is a set of higher-level libraries for building client-server web applications with Ocsigen (Js_of_ocaml and Eliom). It provides modules for user management (session management, registration, activation keys, ...), managing groups of users, displaying tips, and easily sending notifications to the users. Ocsigen Start comes with an eliom-distillery template for an app with a database, user management, and session management. This template is intended to serve as a basis for quickly building the Minimum Viable Product for web applications with users. The goal is to enable the programmer to concentrate on the core of the app, and not on user management.

---
* Homepage: https://ocsigen.org/ocsigen-start/
* Source repo: git+https://github.com/ocsigen/ocsigen-start.git
* Bug tracker: https://github.com/ocsigen/ocsigen-start/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/ocsigen-start/ocsigen-start.1.6.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+maintainer: "dev@ocsigen.org"
+synopsis: "An Eliom application skeleton ready to use to build your own application with users, (pre)registration, notifications, etc"
+description: "Ocsigen Start is a set of higher-level libraries for building client-server web applications with Ocsigen (Js_of_ocaml and Eliom). It provides modules for user management (session management, registration, activation keys, ...), managing groups of users, displaying tips, and easily sending notifications to the users. Ocsigen Start comes with an eliom-distillery template for an app with a database, user management, and session management. This template is intended to serve as a basis for quickly building the Minimum Viable Product for web applications with users. The goal is to enable the programmer to concentrate on the core of the app, and not on user management."
+homepage: "https://ocsigen.org/ocsigen-start/"
+bug-reports: "https://github.com/ocsigen/ocsigen-start/issues"
+dev-repo: "git+https://github.com/ocsigen/ocsigen-start.git"
+license: "LGPL-2.1 with OCaml linking exception"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
+remove: [ make "uninstall" ]
```
Comment (kit-ty-kate) [inline packages/ocsigen-start/ocsigen-start.1.6.0/opam]:
```suggestion
```
The `remove` field should be unnecessary with opam2 if you're not doing anything else other than removing the files you installed.

**Comment by mseri** [comment]
The two travis CI failures are related to camlp4, while the system switch one is expected, I am reruniing the other as we have been fixing camlp4 related issues in the last weeks and maybe there are some more to deal with still.

File: packages/ocsigen-start/ocsigen-start.1.6.0/opam (hunk 1 above)
Comment (jrochel) [reply packages/ocsigen-start/ocsigen-start.1.6.0/opam]: Done.

**Comment by kit-ty-kate** [comment]
Failures look unrelated. Thanks!

**Pull request merged**